test/helpers/kubectl.go

```diff
@@ -302,6 +302,49 @@ func (kub *Kubectl) ManifestGet(manifestFilename string) string {
 	return fmt.Sprintf("%s/k8sT/manifests/%s", BasePath, manifestFilename)
 }
 
+// MicroscopeStart installs (if it is not installed) a new microscope pod,
+// waits until pod is ready, and runs microscope in background. It returns an
+// error in the case where microscope cannot be installed, or it is not ready after
+// a timeout. Also it returns a callback function to stop the monitor and save
+// the output to `helpers.monitorLogFileName` file.
+func (kub *Kubectl) MicroscopeStart() (error, func() error) {
```

Comment:
You're returning three types here but only are returning two in the body of this function. How is this compiling?

Reply:
No, it returns only two arguments, the first one is the `error` and the second one is a callback function that returns an `error`. The definition is correct.

Reply:
Ah! Makes much more sense now ;) thanks.